meson.build

```diff
@@ -1,5 +1,5 @@
 project('watermark-extension',
-  version: '3.8.0',
+  version: '3.9.0rc1',
```

Comment:
I wouldn't bother changing this now as well.

Reply:
So you suggest keeping it numbered as 3.8, even though it does not run on eos3.8? That sounds wrong.

Reply:
Whatever. The lobby for brokenness is strong here.

Reply:
Drive by comment. I think if you want to have a version number for this extension that's driven by another component, you should follow the gnome-shell version. That's a much stronger association anyways since you can only run the extension on specific shell versions, right? I can't think of any other components we develop that uses our OS version as their own version.

Reply:
It was tied to Endless OS' version, because it doesn't always depend on GNOME Shell, but on Endless OS.

Reply:
Yeah, I was thinking the same as @dbnicholson here, maybe we could bump it to 3.38 on the final rebase?